rust/pact_matching_ffi/Cargo.toml

```diff
@@ -22,4 +22,4 @@ log = "0.4.8"
 serde_json = "1.0.51"
 
 [lib]
-crate-type = ["cdylib", "staticlib", "rlib"]
```

Comment:
This change was from my initial efforts to work on this patch, and honestly at this point I don't recall why I removed `staticlib` and `rlib`. `rlib` is used for linking to Rust code, and given that this is FFI is probably not that useful (they can just use `pact_matching` directly), but `staticlib` is good and the `CMake` config could be updated to support static linking as well as dynamic.

Reply:
Of course, I say that, but I haven't investigated what it would take to actually do in CMake. Perhaps for now it's good to keep things simple and focus on dynamic linking, and make the addition of static linking a new issue that can be tackled later.

Reply:
An investigation this morning tells me that providing both static and dynamic linking options with CMake has some complexity, so I'm going to punt on it and come back to it later.